web/src/user_guide/Page.svelte

```diff
@@ -0,0 +1,518 @@
+<script lang="ts">
+  import "@picocss/pico/css/pico.conditional.jade.min.css";
+  import Toc from "svelte-toc";
+  import About from "../About.svelte";
+  import image01 from "./image01.webp?url";
+  import image02 from "./image02.webp?url";
+  import image03 from "./image03.webp?url";
+  import image04 from "./image04.webp?url";
+  import image05 from "./image05.webp?url";
+  import image06 from "./image06.webp?url";
+  import image07 from "./image07.webp?url";
+  import image08 from "./image08.webp?url";
+  import image09 from "./image09.webp?url";
+  import image10 from "./image10.webp?url";
+  import image11 from "./image11.webp?url";
+  import image12 from "./image12.webp?url";
+  import image13 from "./image13.webp?url";
+  import image14 from "./image14.webp?url";
+  import image15 from "./image15.webp?url";
+  import image16 from "./image16.webp?url";
+  import image17 from "./image17.webp?url";
+  import image18 from "./image18.webp?url";
+  import image19 from "./image19.webp?url";
+  import image20 from "./image20.webp?url";
+  import image21 from "./image21.webp?url";
+  import image22 from "./image22.webp?url";
+  import image23 from "./image23.webp?url";
+  import image24 from "./image24.webp?url";
+  import image25 from "./image25.webp?url";
+  import image26 from "./image26.webp?url";
+  import image27 from "./image27.webp?url";
+  import image28 from "./image28.webp?url";
+  import image29 from "./image29.webp?url";
+  import image30 from "./image30.webp?url";
+  import image31 from "./image31.webp?url";
+</script>
+
+<div class="container">
+  <div class="contents">
+    <h1>User guide for the Connected Neighbourhood Tool</h1>
```

Comment:
I thought about including a link back to the tool, but it'd be a small bit of plumbing to remember if the user was on global/England/CNT. Since it opens in a new tab, hopefully people know how to switch back.